crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -838,16 +839,23 @@ def upgrade_controller_clusters
 
         return upgrade_controllers_disruptive if upgrade_mode == :normal
 
-        network_node = ::Node.find(
-          "pacemaker_founder:true AND " \
+        network_founders = ::Node.find(
+          "pacemaker_founder:* AND " \
```

Comment:
Iit should be enough to exclude `pacemaker_founder:*` bit from here

Reply:
dont we need to find all `neutron-network` nodes that are applied into clusters?

Reply:
True. So this "pacemaker_founder:*" acts as an indication that it is a cluster member. Hm, that works, but it's not very readable.

Reply:
yeah, might be easier to search for `run_list_map:pacemaker-cluster-member` so we know that its part of a cluster

Reply:
updated to use that instead and changed var to make it clearer